ozaria/site/components/play/PageIntroLevel/index.vue

```diff
@@ -129,23 +124,40 @@
           this.currentIndex++
           if (this.currentIndex < this.introContent.length) { // increment current content
             this.currentContent = this.introContent[this.currentIndex]
+            this.setCurrentContentId(this.currentContent)
           } else { // whole intro content completed
             await this.setIntroLevelComplete()
-            await this.fetchNextLevel()
-            const link = this.fetchNextLevelLink()
-            if (link && !application.testing) {
-              return application.router.navigate(link, { trigger: true })
+            if ((this.campaignData || {}).levels) {
+              await this.fetchNextLevel()
+              const link = this.fetchNextLevelLink()
+              if (link && !application.testing) {
+                return application.router.navigate(link, { trigger: true })
+              }
             }
           }
         },
+        setCurrentContentId: function (content) {
```

Comment:
A tiny optional nitpick around styling. To decrease the amount of nesting (if that's valuable). Could the non object flow happen first:

```
if (...non object...) {
  this.currentContentId = content.contentId
  return
}
// Object flow.
```

Just my opinion and not necessary.